src/model/parseModelDefinition.ts

```diff
@@ -69,13 +70,25 @@ function deepParseModelDefinition<Dictionary extends ModelDictionary>(
       continue
     }
 
+    if (value instanceof NullableProperty) {
+      // Add nullable properties to the same list as regular properties
+      result.properties.push(propertyPath)
```

Comment:
Once again, a great example of how extending the `Relation` class for `NullableRelation` would allow us to remove this branch altogether.